partner_firstname/models/base_config_settings.py

```diff
@@ -56,9 +58,11 @@ def _onchange_partner_names_order(self):
         self.partner_names_order_changed = self._compute_names_order_changed()
```

Comment:
This assignment already occurs in method `_compute_names_order_changed`. As a consequence, it is failing to hide the button "Recalculate names". If you just call to `self._compute_names_order_changed()` it works.